avocado/core/output.py

```diff
@@ -608,19 +608,23 @@ class LoggingFile(object):
     """
 
     def __init__(self, prefix='', level=logging.DEBUG,
-                 logger=[logging.getLogger()]):
+                 loggers=None):
```

Comment:
just between the two of us, yes. But don't tell it out loud otherwise I'll have to split the patch ... 😄